packages/p2p-v2/src/components/Avatar/Avatar.tsx

```diff
@@ -0,0 +1,14 @@
+import React, { memo } from 'react';
+import './Avatar.scss';
+
+type TAvatarProps = {
+    name: string;
+};
+
+export const getShortNickname = (nickname: string): string => nickname?.substring(0, 2).toUpperCase();
+
+const Avatar = ({ name }: TAvatarProps) => {
```

Comment:
`const Avatar = ({ name } : TAvatarProps) => ( <div className='p2p-v2..... >...)`

Reply:
Noted, I can fix it in the next PR I'm working on right now, cuz I need this profile content for constructing the profile page